applications/dashboard/locale/en.php

```diff
@@ -167,6 +167,7 @@ function formatPossessive($word) {
 $Definition['Test Email Message'] = '<p>This is a test email message.</p>'.
     '<p>You can configure the appearance of your forum\'s emails by navigating to the Email page in the dashboard.</p>';
 $Definition['oauth2Instructions'] = '<p>Configure your forum to connect with an OAuth2 application by putting your unique Client ID, Client Secret, and required endpoints. You will probably need to provide your SSO application with an allowed callback URL, in part, to validate requests. The callback url for this forum is <code>%s</code></p>';
-
+$Definition['External link alert details'] = 'Alert users if they click a link in a post that will lead them away from the forum. 
```

Comment:
Your key here doesn't have the same voice or meaning as the full string. Wouldn't something like "Alert users about external links." be better? Also note that if the translation string is a sentence then the key should be a sentence.